docs/csharp/quick-starts/hello-world.yml

```diff
@@ -0,0 +1,176 @@
+    > concatenate strings together. Experiment in the interactive window. Write
+    > some more messages yourself. Build strings using the `+` operator.
+- title: Do more with strings
+  durationInMinutes: 4
+  content: |
+    You've been using a **method**, `Console.WriteLine` to print messages. A **method** is a block of code that implements some action. It has a name, so you can access it. 
+    
+    The `string` type has many useful methods. Let's explore the `Trim` method, and related methods `TrimStart` and `TrimEnd`. These methods remove leading and trailing spaces:
+
+    ```csharp
+    string message = "      Hello World!       ";
+    Console.WriteLine("[" + message + "]");
+
+    string trimmedMessage = message.TrimStart();
+    Console.WriteLine("[" + trimmedMessage + "]");
+
+    trimmedMessage = message.TrimEnd();
+    Console.WriteLine("[" + trimmedMessage + "]");
+
+    trimmedMessage = message.Trim();
+    Console.WriteLine("[" + trimmedMessage + "]");
+    ```
+
+    This sample reinforces a couple of important concepts for working with strings. The methods that manipulate strings return new string objects rather than making modifications in place. You can see that each call to any of the Trim methods returns a new string, but does not change the original message.
+
+    Other methods perform more general edits on a string. You've probably used a search and replace command in an editor or word processor. The `Replace` method does something similar in a `string`. It searches for a substring, and replaces it with different text. The `Replace` method takes two **parameters**. These are the strings between the parentheses. The first string is the text to search for. The second string is the text to replace it with. Try it for yourself. Add this code above:
+
+    ```csharp
+    string message = "Hello World!";
+    Console.WriteLine(message);
+    message = message.Replace("Hello", "Greetings");
+    Console.WriteLine(message);
+    ```
+
+    Two other useful methods make a string ALL CAPS or all lower case. Try the following code:
+
+    ```csharp
+    string message = "Hello World!";
+    Console.WriteLine(message);
+    Console.WriteLine(message.ToUpper());
+    Console.WriteLine(message.ToLower());
+    ```
+- title: Search strings
+  durationInMinutes: 4
+  content: |
+    The other part of a *search and replace* operation is to find text in a
+    string. You can use the the `Contains` method for searching. It
+    tells you if a string contains a substring inside it. Try the following
+    code to explore `Contains`:
+
+    ```csharp
+    string message = "You say goodbye, and I say hello.";
+    Console.WriteLine(message.Contains("goodbye"));
+    Console.WriteLine(message.Contains("greetings"));
+    ```
+
+    The `Contains` method returns a **boolean** value which tells you if the
+    string you sought was found. A **boolean** stores either a `true` or a
```

Comment:
wondering if sought is a hard term for ESL speakers. Maybe replace "you sought" with "you were searching for"?